script/tool/test/xctest_command_test.dart

```diff
@@ -157,19 +201,14 @@ void main() {
       });
 
       test('running with correct destination', () async {
-        final Directory pluginDirectory =
-            createFakePlugin('plugin', packagesDir, extraFiles: <String>[
-          'example/test',
-        ], platformSupport: <String, PlatformSupport>{
+        final Directory pluginDirectory = createFakePlugin(
+            'plugin', packagesDir, platformSupport: <String, PlatformSupport>{
           kPlatformIos: PlatformSupport.inline
         });
 
         final Directory pluginExampleDirectory =
             pluginDirectory.childDirectory('example');
 
-        processRunner.processToReturn = MockProcess.succeeding();
-        processRunner.resultStdout =
-            '{"project":{"targets":["bar_scheme", "foo_scheme"]}}';
```

Comment:
Opportunistic cleanup; I'm assuming that this is from an earlier version of `xctest_command`, because nothing currently uses this output.